sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -15,6 +15,7 @@ public class ClientTestFixtureAttribute : NUnitAttribute, IFixtureBuilder2, IPre
     {
         public static readonly string SyncOnlyKey = "SyncOnly";
         public static readonly string RecordingDirectorySuffixKey = "RecordingDirectory";
+        public static readonly string UseLatestServiceVersionKey = "AZURE_USE_LATEST_SERVICE_VERSION";
```

Comment:
Can we call this `AZURE_ONLY_TEST_LATEST_SERVICE_VERSION` to be clearer about what it's doing?